examples/cube.py

```diff
@@ -18,6 +18,19 @@
 camera = gfx.PerspectiveCamera(70, 16 / 9)
 camera.position.z = 400
 
+m1 = cube.material
+m2 = gfx.MeshPhongMaterial(color="#ff6699")
+m3 = gfx.MeshBasicMaterial(color="#11ff99")
+
+p1 = cube.geometry.positions
+p2 = gfx.Buffer(cube.geometry.positions.data * 0.7)
+
+g1 = cube.geometry
+g2 = gfx.Geometry()
+g2.positions = p2
+g2.normals = g1.normals
+g2.indices = g1.indices
```

Comment:
what is the point of these changes? they don't seem to be added into the scene at all?

Reply:
I think this is me trying things out. Will clean this up.